acados/ocp_nlp/ocp_nlp_constraints_bghp.c

```diff
@@ -1104,6 +1111,8 @@ void ocp_nlp_constraints_bghp_config_initialize_default(void *config_)
     config->memory_set_lam_ptr = &ocp_nlp_constraints_bghp_memory_set_lam_ptr;
     config->memory_set_DCt_ptr = &ocp_nlp_constraints_bghp_memory_set_DCt_ptr;
     config->memory_set_RSQrq_ptr = &ocp_nlp_constraints_bghp_memory_set_RSQrq_ptr;
+    config->memory_set_z_alg_ptr = &ocp_nlp_constraints_bgh_memory_set_z_alg_ptr;
```

Comment:
`ocp_nlp_constraints_bghP_memory_set_z_alg_pt` plus next line plus adding the functions (which either are implemented properly or print-and-exit)

Reply:
addressed here 3a47d76a0 and here 3a47d76a0.